gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -38,23 +38,23 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	//k8s.io/kubernetes/pkg/volume
-	//storage "k8s.io/client-go/pkg/apis/storage/v1beta1"
```

Comment:
Ok. The github review feature is not very helpful for per-patch comments... My last comment that requested splitting of a patch to reflect what the commit message acually describes was meant for this commit ...